portal/server/utils/destroy-session-and-redirect-to-start.js

```diff
@@ -0,0 +1,18 @@
+const { PORTAL_URL } = require('../constants');
+
+/**
+ * Destroys a user's session and redirects them to the start page.
+ * @param {Object} req
+ * @param {Object} res
+ * @returns {void}
+ */
+function destroySessionAndRedirectToStart(req, res) {
+  const startPageRedirect = Boolean(process.env.START_PAGE_REDIRECT);
```

Comment:
A note on any other env variables which needs to be added to micro-services as part of 2FA please in PR description will be good.